ldi-core/ldes-client/tree-node-supplier/src/test/java/ldes/client/performance/FragmentSize.java

```diff
@@ -6,7 +6,10 @@ public enum FragmentSize {
 	TEN("http://localhost:10101/mobility-hindrances/pagination10?pageNumber=1"),
 	HUNDRED("http://localhost:10101/mobility-hindrances/pagination100?pageNumber=1"),
 	TWOFIFTY("http://localhost:10101/mobility-hindrances/pagination250?pageNumber=1"),
-	EXT("http://localhost:8080/mobility-hindrances/pagination250?pageNumber=1");
+	EXT("http://localhost:8080/mobility-hindrances/pagination250?pageNumber=1"),
+	EXT_TWOFIFTY("http://localhost:8080/observations/by-page?pageNumber=1"),
```

Comment:
Can you give these EXT views consistent names? Now you have 'pagination250', 'by-page', 'pagination' and 'by-page1000'..